### ISSUE TITLE
Upon install I was immediately the target of a ransomware attack

### ISSUE THREAD

**Issue by oliverfoster27** [GlennPatrickMurphy/pm-gpt#1]
Pls help or give me bitcoin so I can get my life back

**Comment by GlennPatrickMurphy** [comment]
This is not a bug, the product is working as expected.

**Comment by mayuranganesathas** [comment]
is this safe to install?

**Comment by oliverfoster27** [comment]
No. The assailant stole my identity and all my money. The Government has refused my claim so I am now forced to go by the name "Darryl Schrumph". Horrible experience. 2/5 stars.
